easy_rec/python/input/parquet_input.py

```diff
@@ -0,0 +1,336 @@
+# -*- encoding:utf-8 -*-
+# Copyright (c) Alibaba, Inc. and its affiliates.
+import logging
+import multiprocessing
+import queue
+# import threading
+import time
+
+# import numpy as np
+# import pandas as pd
+import tensorflow as tf
+# from tensorflow.python.framework import ops
+from tensorflow.python.ops import array_ops
+# from tensorflow.python.ops import logging_ops
+# from tensorflow.python.ops import math_ops
+from tensorflow.python.platform import gfile
+
+from easy_rec.python.compat import queues
+from easy_rec.python.input import load_parquet
+from easy_rec.python.input.input import Input
+
+
+class ParquetInput(Input):
```

Comment:
add docstring for choice between ParquetInput and ParquetInputV2

Reply:
add in dataset.proto file